### ISSUE TITLE
Failed to tweet; Malformed UTF-8 characters, possibly incorrectly encoded

### ISSUE THREAD

**Issue by cartelvid** [10up/autoshare-for-twitter#271]
### Describe the bug

When i try "Tweet Now" i see this error: Failed to tweet; Malformed UTF-8 characters, possibly incorrectly encoded

And Callback Url is here: https://www.javuln.com/wp-admin/admin-post.php?action=authoshare_authorize_callback

"Something went wrong. Please try again." error is in here.

Dont hack me please :D i want to use your plugin and i hope it will be greater than now.

### Steps to Reproduce

1. "Tweet Now" and show me error.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by iamdharmesh** [comment]
Hi @cartelvid, 

Thank you for reporting the issue.

Based on the error it seems the error is due to malformed UTF-8 characters in the tweet body. Are you facing this error with every post/page or on a specific post only? Also, could you please help to provide the tweet message or post title in which you get this error? It will be helpful to identify the issue you are facing.

Thank you.

**Comment by cartelvid** [comment]
Not a specific post only. Its on every post/page. I think you should make a
support to all characters.
Thank you.

Dharmesh Patel ***@***.***>, 25 Ağu 2023 Cum, 16:21 tarihinde
şunu yazdı:

**Comment by Sidsector9** [comment]
@iamdharmesh I tested with some non UTF-8 characters but was not reproduce the issue.

I suspect this to be the error causing line:

https://github.com/abraham/twitteroauth/blob/8eb8a3bec3f7ea2850ad6d3fd4e3450423b30136/src/TwitterOAuth.php#L817

@cartelvid can you share the text you're trying to tweet?

**Comment by cartelvid** [comment]
Sure, here it is. " Failed to tweet; Malformed UTF-8 characters, possibly
incorrectly encoded"

Siddharth Thevaril ***@***.***>, 7 Eyl 2023 Per, 10:48
tarihinde şunu yazdı:

**Comment by iamdharmesh** [comment]
> Sure, here it is. " Failed to tweet; Malformed UTF-8 characters, possibly
incorrectly encoded"

@cartelvid Could you please provide a post title or custom tweet message that you are trying to tweet? This is the error message.

Thank you.

**Comment by cartelvid** [comment]
"หนังไทย" its thai language

Dharmesh Patel ***@***.***>, 8 Eyl 2023 Cum, 18:10 tarihinde
şunu yazdı:

**Comment by jeffpaul** [comment]
@iamdharmesh is this actionable?

**Comment by iamdharmesh** [comment]
No, this is not reproducible on our end. I am going to close this, but please feel free to reopen it if we receive other reports. Thanks!
